scipy/stats/_multivariate.py

```diff
@@ -1421,8 +1421,9 @@ def mean(self, alpha):
 
         Returns
         -------
-        mu : scalar
-            Mean of the Dirichlet distribution
+        mu : ndarray
```

Comment:
Annoyingly ``var`` doesn't go through ``_squeeze_output``, so does return an array of size 1. I suggest fixing that up so it does return a scalar.